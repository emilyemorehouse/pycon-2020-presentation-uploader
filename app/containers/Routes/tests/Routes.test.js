import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

import { Routes } from '..'

const renderer = new ShallowRenderer()

describe('Routes', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<Routes />)

    const renderedOutput = renderer.getRenderOutput()

    expect(renderedOutput).toMatchSnapshot()
  })
})
