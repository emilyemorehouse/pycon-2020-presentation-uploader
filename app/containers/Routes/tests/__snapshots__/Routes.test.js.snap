// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render and match the snapshot 1`] = `
<React.Fragment>
  <HelmetWrapper
    defaultTitle="React.js Boilerplate"
    defer={true}
    encodeSpecialCharacters={true}
    titleTemplate="%s - React.js Boilerplate"
  >
    <meta
      content="A React.js Boilerplate application"
      name="description"
    />
  </HelmetWrapper>
  <Memo(wrappedComponent) />
  <Routes__Container>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "compare": null,
            "displayName": "",
            "propTypes": Object {
              "history": [Function],
              "location": [Function],
            },
            "type": [Function],
          }
        }
        path="/confirm-account"
      />
      <Memo(wrappedComponent)
        component={[Function]}
        exact={true}
        path="/login"
      />
      <Memo(wrappedComponent)
        component={[Function]}
        exact={true}
        path="/register"
      />
      <Memo(wrappedComponent)
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "compare": null,
            "displayName": "",
            "type": [Function],
          }
        }
        exact={true}
        path="/dashboard"
      />
      <Memo(wrappedComponent)
        component={[Function]}
        path="/features"
      />
      <Route
        component={[Function]}
        path=""
      />
    </Switch>
  </Routes__Container>
  <Memo(GlobalStyleComponent) />
</React.Fragment>
`;
